api/types/src/convert.rs

```diff
@@ -1064,24 +1064,33 @@ impl<'a, S: StateView> MoveConverter<'a, S> {
                 location,
                 function,
                 code_offset,
+                message,
             } => {
                 let func_name = match location {
                     AbortLocation::Module(module_id) => self
                         .explain_function_index(module_id, function)
                         .map(|name| format!("{}::{}", abort_location_to_str(location), name))
                         .unwrap_or_else(|_| {
                             format!(
-                                "{}::<#{} function>",
+                                "{}::<#{} function>, {:?}",
                                 abort_location_to_str(location),
-                                function
+                                function,
+                                message
                             )
                         }),
                     AbortLocation::Script => "script".to_owned(),
                 };
-                format!(
-                    "Execution failed in {} at code offset {}",
-                    func_name, code_offset
-                )
+                if let Some(message) = message {
```

Comment:
Looks like `txn_aux_data` (the param to this function) potentially contains `VMErrorDetail`, which we may also want to display here?

Reply:
I do not think we populate `txn_aux_data` anymore, now that AIP to store debug info on the side is removed :(